src/main/java/ca/corefacility/bioinformatics/irida/ria/web/sessionAttrs/Cart.java

```diff
@@ -0,0 +1,73 @@
+package ca.corefacility.bioinformatics.irida.ria.web.sessionAttrs;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Session object to hold samples that are currently in the cart.
+ */
+public class Cart extends HashMap<Long, HashSet<Long>> {
+
+	public int add(Long projectId, List<Long> sampleIds) {
```

Comment:
Method comment

Reply:
Added in ff327b3dfaf6c5db4002fe5a0c32ffbf4a7512c7